### ISSUE TITLE
Load weights

### ISSUE THREAD

**Issue by MatesdeSilvia** [AISoltani/Mask-RCNN-Tensorflow2.7.0-Keras2.7.0#1]
@AISoltani 
I have used mrcnn_predict.py and I have problems with the load weights function.
When I put `model.load_weights(filepath="mask_rcnn_coco.h5",
                  by_name=True)`

the result is 
![111](https://github.com/AISoltani/Mask-RCNN-Tensorflow2.7.0-Keras2.7.0/assets/148212950/9ce19096-cecd-4c34-8813-686693c5450a)

however, when I use the function` tf.keras.Model.load_weights(model.keras_model, "mask_rcnn_coco.h5" , by_name=True)` 

the result is correct:
![222](https://github.com/AISoltani/Mask-RCNN-Tensorflow2.7.0-Keras2.7.0/assets/148212950/f0bc492a-0323-4582-8306-865d3b3bd5d2)

I need to train a new class using the coco weights, so I would need to use the function `model.load_weights(filepath="mask_rcnn_coco.h5",
                  by_name=True) `

How could I solve it?
